### ISSUE TITLE
✨ Polishing: Inconsistent dialogs / modal and other problems

### ISSUE THREAD

**Issue by SebastianStehle** [novuhq/novu#2746]
### 📜 Description

It would be great to make the dialogs more consistent, e.g.

* Consistent background colors
* Consistent close buttons
* Consistent button placements? (e.g. always right?)
* Only one scrollbar (see last screenshot)

Sample samples:

![image](https://user-images.githubusercontent.com/1236435/218336121-cf75ba89-e28a-413c-80b4-5401ed84bf95.png)
![image](https://user-images.githubusercontent.com/1236435/218336126-47b067b5-296c-4460-bf81-9462c5d6e2e7.png)
![image](https://user-images.githubusercontent.com/1236435/218336142-bbfbf6be-ac17-4b12-8d0e-f91c1a78a35f.png)
![image](https://user-images.githubusercontent.com/1236435/218336271-1043ef19-068e-4334-897c-2bd450ce272b.png)
![image](https://user-images.githubusercontent.com/1236435/218336233-855da8c5-882f-4001-8cdd-77c76d428357.png)

### 👀 Have you spent some time to check if this bug has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Contributing Guidelines?

- [X] I have read the [Contributing Guidelines](https://github.com/novuhq/novu/blob/main/CONTRIBUTING.md)

**Comment by oba2311** [comment]
Great! adding the label.

**Comment by oba2311** [comment]
@prosper adding here next cycle meanwhile - WDYT as per this timeframe to get it done?

**Comment by scopsy** [comment]
@oba2311 we need the actual design for this to be able to work on this. This feature is not ready for work yet.

**Comment by oba2311** [comment]
Yes agreed, @unicodeveloper  please make sure no one is working on it till @nikolay  get to it (which will not happen this week).

**Comment by oba2311** [comment]
@unicodeveloper  BTW I'm adding the blocked by so you will know when it is ready 😄

**Comment by unicodeveloper** [comment]
Alright. Thanks @oba2311
